Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1294,6 +1294,7 @@ extension BrowserViewController: WKNavigationDelegate {
             if !ReaderModeUtils.isReaderModeURL(url) {
                 urlBar.updateReaderModeState(ReaderModeState.Unavailable)
             }
+            hideReaderModeBar(animated: false)
```

Comment:
Can this go inside the if statement above? I can't think of any situations where we'd directly load a reader page from a reader page, but it can't hurt to have it in the same place we do the other reader mode update.